src/components/CreateMatch/CreateMatch.js

```diff
@@ -109,23 +141,32 @@ class CreateMatch extends Component {
                         <img className="match-name__shield" src={shield} alt="escudo del equipo" />
                     </span>
                     <img className="match-main__img" src={soccerField} alt="campo de juego" />
-
+                    {showPlayersField ? <Players colorSelected={colorSelected} /> : ''}
```

Comment:
if you are not returning anything on the false case, don't use the ternary, instead use this `showPlayersField && <Players colorSelected={colorSelected} />`. Review all your code and verify that in those cases you always use &&